apps/web/components/booking/BookingListItem.tsx

```diff
@@ -353,6 +354,25 @@ function BookingListItem(booking: BookingItemProps) {
                   attendees={booking.attendees}
                 />
               </div>
+              {!isPending && booking.location && (
+                <Link href={bookingLink} className="text-sm leading-6 text-blue-400 hover:underline">
```

Comment:
the booking link is not the video link.

clicking the "join" button opens the booking details, not the location URL

Reply:
I think we can get the video URL by doing something as follows in the booking get handler

https://github.com/calcom/cal.com/pull/11942/commits/cecd2ae537d0e601df7a8c9d6deb3b3719792e32

(just a git diff for reference) 

you should be able to access booking.metadata.videoUrl (I havent tested but the data is there in the DB :P)

Reply:
working on it rn.

Reply:
Seems like the booking prop which is being used on this page isn't having a metadata.
<img width="1440" alt="Screenshot 2023-10-17 at 6 10 51 PM" src="https://github.com/calcom/cal.com/assets/94699055/4d6ca034-67e1-4599-abbd-48f3a549af37">

Reply:
tried booking.refrences[0].meetingUrl, it's giving meeting links for google meet and zoom video but not for cal video or a event that has external meeting link.

Reply:
Hmmm - this appears to be along the right lines https://github.com/calcom/cal.com/blob/75d7d2f17291e7b1e137839b1de3eac998ecfa0d/apps/web/pages/booking/%5Buid%5D.tsx#L117 on how to get this info. Looks like its a bit complicated to get but deffo possible if you follow it through. 

There is also this fn which might help https://github.com/calcom/cal.com/blob/75d7d2f17291e7b1e137839b1de3eac998ecfa0d/packages/app-store/locations.ts#L392

Reply:
yes I was currently doing that. Just need more time.

Reply:
@sean-brydon thanks for the help. Seems like I was just being dumb and I thought cal video was a default app but then I found daily.co api integration. The meeting links now work fine for both the apps one and external meeting link types.